core/src/main/java/com/opensymphony/xwork2/validator/AnnotationActionValidatorManager.java

```diff
@@ -120,10 +66,20 @@ protected String buildValidatorKey(Class clazz, String context) {
         } else {
             sb.append(context);
         }
-
         return sb.toString();
     }
 
+    @Override
+    protected Validator getValidatorFromValidatorConfig(ValidatorConfig config, ValueStack stack) {
+        Validator validator = validatorFactory.getValidator(
+                new ValidatorConfig.Builder(config)
+                        .removeParam("methodName")
```

Comment:
Is this override necessary? I'm not completely clear on what it's achieving. Would be nice to delete it if it doesn't do anything.

Reply:
Looks like it does nothing right now, you can try to remove it.

Reply:
Done :)